### ISSUE TITLE
[DOC] Proofreading of the OpenMS documentation

### ISSUE THREAD

**Issue by enetz** [OpenMS/OpenMS#6050]
The OpenMS documentation is being written by many people in chunks that are usually written once together with a class and then rarely double checked or proofread. Some typos, grammatical errors and suboptimal formulations remain in the documentation and should be fixed.

To start fixing this, build OpenMS with the documentation. Read the built doc in OpenMS_build/doc/index.html and look for typos and other grammatical errors and improve readability and completeness of descriptions etc. by editing the doxygen source code in OpenMS/doc and doxygen descriptions in C++ class files.
Rebuild and double check the updated documentation locally before submitting a pull request with your fixes.

**Comment by timosachsenberg** [comment]
Is there an easy way to find undocumented public methods? And Maybe an automated way to prioritize them by usage in tools and library?

**Comment by greengypsy** [comment]
Yes, there is a flag you need to turn on in the Doxyfile (WARN_IF_UNDOCUMENTED).
<img width="581" alt="Screen Shot 2022-06-06 at 9 16 26 pm" src="https://user-images.githubusercontent.com/31978630/172154671-3475c372-d1ca-4175-be6c-05a2cf92e1e5.png">

**Comment by greengypsy** [comment]
I can't find any way to filter these to only public ones. A script might need to be created to filter the results are the doxyfile is run.
